views/swap/input/select-token.tsx

```diff
@@ -44,6 +44,8 @@ const SelectToken: FC<InputProps> = ({ label }) => {
   });
 
   const onSelect = async (metadata: AssetMetadata) => {
+    if (metadata.symbol === opposite.symbol) return;
```

Comment:
Instead of using symbol, be like someone can create it own POPCAT and if we have 2 diff POPCAT he would be able to trade, basically we need to cook something even better, I mean use the `FA_ADDRRESS` and `COIN_TYPE_TO_FA` constants to help you